server.js

```diff
@@ -64,10 +63,25 @@ app.prepare().then(() => {
       // path /(help|chat) -> Discord chat invite
       res.writeHead(301, { Location: 'https://discordapp.com/invite/dvwXA2N' })
       res.end()
+    } else if (/^\/(docs|documentation)(\/.*)?$/i.test(pathname)) {
+      // path /docs... or /documentation... -> /doc...
+      res.writeHead(301, {
+        Location: req.url.replace(/\/(docs|documentation)/i, '/doc')
+      })
+      res.end()
     } else if (/^\/doc\/commands-reference(\/.*)?/.test(pathname)) {
       // path /doc/commands-reference... -> /doc/command-reference...
       res.writeHead(301, {
-        Location: req.url.replace('commands-reference', 'command-reference')
+        Location: req.url.replace(
```

Comment:
why do we need this change?

Reply:
1. A bit of consistency: the other `req.url.replace` calls use the full path from root `/`.
2. It makes it less probable that the string to replace is repeated in the URL (already unlikely with just `commands-reference`, but much less with the whole `/doc/commands-reference`.

Reply:
p.s. I just tested #2 here and it's not the case. So just consistency with all the other `req.url.replace` in the file.

Reply:
p.s. did a similar change in https://github.com/iterative/dvc.org/pull/882/files#diff-78c12f5adc1848d13b1c6f07055d996eL93-R104